### DISCUSSION TITLE
Problemi con aggiornamento 1.19.0 di zigbee2mqtt

### DISCUSSION THREAD

**Discussion by gualo1983** [indomus/forum#387] [ZigBee, Z-Wave, Thread, Matter]
ho aggiornato il container docker di zigbee2mqtt con il classico comando docker-compose pull zigbee2mqtt
ho fatto il build e cancellato la vecchia immagine del container.
dopo che tutti i passaggi sono andati a buon termine come sempre però l'interfaccia web non funzionava.
ho provato a dare il comando docker container logs zigbee2mqtt -f ed ecco cosa mi esce scritto:

> Using '/app/data' as data directory
>  Fatal error in , line 0
>  unreachable code
> FailureMessage Object: 0x7ef6b25c

il messaggio si ripete svariate volte cambiando solo il codice 0x7....
il file di configurazione di docker compose non lo ho toccato 
 ` zigbee2mqtt:`
`    container_name: zigbee2mqtt`
`    image: koenkk/zigbee2mqtt`
`    volumes:`
`    - "/home/pi/zigbee2mqtt:/app/data"`
`    devices:`
`    - "/dev/ttyACM0"`
`    environment:`
`    - "TZ=Europe/Rome"`
`    network_mode: host`
`    restart: always`
e anche il file di configurazione di zigbee2mqtt è rimasto inalterato
` homeassistant: false`
` permit_join: false`
` mqtt:`
`   base_topic: zigbee2mqtt`
`   server: 'mqtt://localhost'`
`   user: XX`
`   password: XXXXXXXXX`
` serial:`
`   port: /dev/ttyACM0`
`   adapter: deconz`
` advanced:`
`   log_directory: /var/log`
` experimental:`
`   new_api: true`
` frontend:`
`   port: 8080`
` devices:`
`   '0x00158d00042fda4a':`
`     friendly_name: Sensore1`
`   '0x00158d0004a113e1':`
`     friendly_name: Finestra1`
`   '0x00158d000484f834':`
`     friendly_name: Sensore2`
`   '0x00158d0001dde55a':`
`     friendly_name: Sensore3`
`   '0x00158d0001dde5aa':`
`     friendly_name: Sensore4`
`   '0x00158d0001dcfc9f':`
`     friendly_name: Sensore5`

**Comment by gualo1983** [comment]
cercando in github il progetto koenkk/zigbee2mqtt ho trovato che anche altri hanno avuto questo problema.
Per risolvere il programmatore ha creato una versione 1.19.1 per attivarla nel file di configurazione di docker compose ho dovuto mettere koenkk/zigbee2mqtt:latest-dev
facendo questa modifica tutto è partito e anche HA riceve tutti i dati dei sensori.

**Reply by snakuzzo** [reply]
Bah...non saprei.
Ho appena aggiornato alla stable con docker-compose. Tutto ok
